### DISCUSSION TITLE
Explorer quick actions panel

### DISCUSSION THREAD

**Discussion by esjs** [microsoft/vscode-discussions#468] [Extension Development QnA] [answered]
Hey everyone,

Can you please direct me to required setting in VSCode or some plugin to make Explorer panel display quick actions for folder, similar to how it's done on CodeSandbox platform (see attached screenshot) ?
<img width="283" alt="image" src="https://user-images.githubusercontent.com/2673700/219605352-012b6a43-18e7-41fe-b075-253ebe67fd5a.png">

If there is none, do somebody know whether it's possible to create such an extension (link to API documentation which would allow to do this or something on this topic)? 

I've never written extensions for VSCode, but can try to do it if somebody will point me to right direction.

Thanks in advance!

**Comment by gjsjohnmurray** [comment, marked as answer]
I don't think this is currently possible. Extensions can contribute to the context menu a user gets when right-clicking on a file or folder in Explorer, but they can't contribute command buttons alongside items in the tree.
